### ISSUE TITLE
Equivalent Data PreProcessing for complex-valued input

### ISSUE THREAD

**Issue by SantaTitular** [NEGU93/cvnn#44]
Hi again Jose,

Apologies if this questions is not directly code related but it might be relevant considering some [classification examples ](https://complex-valued-neural-networks.readthedocs.io/en/latest/code_examples/cifar10.html) that I saw in your docs. In real-valued NNs we often use data pre-processing (e.g., [skleanr's StandardScaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html) ). In case we have some complex-valued data, e.g., backscatter parameter such as S11 from a VNA (because I think your familiarized with it 😅), should we also normalize it (despite it already being between 0-1)? Also how do we normalize the complex input in these cases, is it the same as using complex batch norm layer before inputting onto the network? 

I previously read your excellent paper ([Impact of PolSAR Pre-Processing...](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=10048504)) but I dont think it discusses this aspect (I'll re-read it still).

Thanks in advance!
Tomás
